mithril-signer/src/runtime/runner.rs

```diff
@@ -38,7 +42,13 @@ pub trait Runner: Send + Sync {
     async fn can_sign_current_epoch(&self) -> StdResult<bool>;
 
     /// Register epoch information
-    async fn inform_epoch_settings(&self, epoch_settings: SignerEpochSettings) -> StdResult<()>;
+    async fn inform_epoch_settings(
+        &self,
+        signer_epoch: Epoch,
```

Comment:
This is more the epoch retrieved from the aggregator than from the signer:
```suggestion
        aggregator_epoch: Epoch,
```

Or better:
```suggestion
        aggregator_signer_registration_epoch: Epoch,
```